<?xml version="1.0"?>
<launch>
	<env name="ROSCONSOLE_CONFIG_FILE" value="$(find c2_hw)/config/c2_logger.config"/>
<rosparam ns="calibrate_pid">
left_pid:  {p: 4.0, i: 0.01, d: 1.5}
right_pid: {p: 4.0, i: 0.01, d: 1.5}
plan:
  - {t: 100, l: 0.25, r: 0.25}
</rosparam>
	<node name="calibrate_pid" pkg="c2_hw" type="calibrate_pid" >
		<remap from="/lwheel_vtarget" to="/c2/hardware/wheels/l_cmd_vel"/>
		<remap from="/rwheel_vtarget" to="/c2/hardware/wheels/r_cmd_vel"/>
		<remap from="/lwheel_state" to="/c2/hardware/wheels/l_vel"/>
		<remap from="/rwheel_state" to="/c2/hardware/wheels/r_vel"/>
	</node>
</launch>
